projects/plugins/jetpack/tools/build-block-manifest.php

```diff
@@ -0,0 +1,91 @@
+<?php // phpcs:ignore Squiz.Commenting.FileComment.MissingPackageTag
+/**
+ * Build script to generate a PHP manifest file containing block metadata from block.json files.
+ * This reduces filesystem reads and JSON parsing at runtime.
+ */
```

Comment:
Why not just add one?
```suggestion
<?php
/**
 * Build script to generate a PHP manifest file containing block metadata from block.json files.
 * This reduces filesystem reads and JSON parsing at runtime.
 *
 * @package automattic/jetpack
 */
```